src/playback/qml/MuseScore/Playback/internal/MixerTitleSection.qml

```diff
@@ -82,10 +82,12 @@ MixerPanelSection {
             hoverEnabled: true
 
             onContainsMouseChanged: {
-                if (mouseArea.containsMouse) {
-                    ui.tooltip.show(root, channelItem.title)
+                var instrumentName = channelItem.title
+                if (mouseArea.containsMouse && instrumentName.length > 14) {
```

Comment:
Hard-coding the number of 14 here is not a very bullet-proof solution: 
- for different fonts (or font sizes), the decision boundary will be at a different amount of characters
- different characters within a font have different widths

It would be better to check whether the text of the label is actually being elided or not. As I said in https://github.com/musescore/MuseScore/pull/19414#issuecomment-1818483882, the label has a property for that: [`truncated`](https://doc.qt.io/qt-5/qml-qtquick-text.html#truncated-prop). To access it, or rather to access _the label_, just give the label an `id`, like with the MouseArea.